src/runtime/components/CommandPalette.vue

```diff
@@ -259,7 +260,7 @@ const groups = computed(() => {
       <ComboboxContent :class="ui.content({ class: props.ui?.content })" :dismissable="false">
         <ComboboxEmpty :class="ui.empty({ class: props.ui?.empty })">
           <slot name="empty" :search-term="searchTerm">
-            {{ searchTerm ? `No results for ${searchTerm}` : 'No results' }}
+            {{ searchTerm ? t('ui.commandPalette.noMatch', { search: searchTerm }) : t('ui.commandPalette.noData') }}
```

Comment:
Why not keep `searchTerm` name?

Reply:
I agree `searchTerm` would be better, will fix it